contrib/bpftrace/gc_all.bt

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bpftrace
+
+BEGIN
+{
+	printf("Tracing Julia GC Times... Hit Ctrl-C to end.\n");
+}
+
+usdt:usr/lib/libjulia-internal.so:julia:gc__begin
+{
+	$now = nsecs;
+	@time[pid] = $now;
+	@start[pid] = $now;
+}
+
+usdt:usr/lib/libjulia-internal.so:julia:gc__stop_the_world
+/@start[pid]/
+{
+	$now = nsecs;
+	@stop_the_world[pid] = hist(($now - @time[pid]) / 1000);
```

Comment:
It could be lovely to expand this and similar names with units, e.g. stop_the_world_ms, gc_total_ms, and gc_phase_ms, so that folks don't have to guess or hunt about? :)

Reply:
And of course, case in point, I get the units wrong... perhaps stop_the_world_usecs, gc_total_usecs, gc_phase_usecs, and finalizer_usecs? :)

Reply:
Good idea! At least for the ones we echo in the end.